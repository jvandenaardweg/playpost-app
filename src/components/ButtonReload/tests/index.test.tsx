import React from 'react';
import { render, RenderAPI } from 'react-native-testing-library';

import { ButtonReload } from '../index';

describe('ButtonReload', () => {

  describe('rendering', () => {
    let wrapper: RenderAPI;

    beforeAll(() => {
      wrapper = render(<ButtonReload onPress={() => {}} />);
    });

    it('should render correctly', () => {
      expect(wrapper.toJSON()).toMatchSnapshot();
    });
  });
});
