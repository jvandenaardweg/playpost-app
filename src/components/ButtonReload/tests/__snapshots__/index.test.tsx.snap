// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonReload rendering should render correctly 1`] = `
<View
  accessible={true}
  clickable={true}
  isTVSelectable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "height": 40,
      "justifyContent": "center",
      "marginRight": 6,
      "opacity": 1,
      "width": 40,
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": "#037DE2",
          "fontSize": 20,
        },
        undefined,
        Object {
          "fontFamily": "Feather",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ï†»
  </Text>
</View>
`;
